src/oic/utils/aes.py

```diff
@@ -109,7 +110,8 @@ def decrypt(key, msg, iv=None, padding="PKCS#7", b64dec=True):
 
     _iv = data[: AES.block_size]
     if iv:
-        assert iv == _iv
+        # doesn't really matter as the decrypt will fail if its wrong
```

Comment:
Than we can probably remove it completely?

Reply:
ok